core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -3062,6 +3063,14 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey AUTHENTICATION_STALE_CHANNEL_PURGE_INTERVAL =
+      new Builder(Name.AUTHENTICATION_STALE_CHANNEL_PURGE_INTERVAL)
+          .setDefaultValue("60min")
+          .setDescription("Interval at which stale channels will be removed from "
+              + "authentication cache.")
```

Comment:
Could you also briefly mention the implications of shorter and longer intervals? This description will be in the documentation, so how/when would you advise users to adjust this?

Reply:
I'd like to address this as part of merge PR if it's OK.